.rubocop.yml

```diff
@@ -62,6 +62,12 @@ Metrics/BlockLength:
     - resource
     - context
     - describe
+    - it
+    - specify
+    - fcontext
+    - fdescribe
+    - fit
+    - fspecify
```

Comment:
1) I think that `it` and `specify` shouldn't be included in this list and, instead, we should be trying to reduce their size. There are already some flagged in the `.rubocop_todo.yml` file. I may be wrong - is there a standard convention?

2) The `f*` versions shouldn't really be checked in so they don't need to be listed here. At least, following (1), perhaps just `fcontext` and `fdescribe` are required.

Reply:
if you see my comment in the commit the length of a specify or it block is already covered by RSpec/ExampleLength. This is just to prevent Metrics/BlockLength cop warnings unnecessarily cluttering the messages.